### DISCUSSION TITLE
How get back embedded parametrize tests? #22913

### DISCUSSION THREAD

**Discussion by Wintreist** [pytest-dev/pytest#11970] [Q&A] [answered]
Hello, I create test using pytest. Before was if test have parametrize mark then his look in testing panel like this:
![image](https://github.com/microsoft/vscode-python/assets/49996562/798dfe62-6bc9-459e-8e13-bf0b062faa42)
Now I updated to python 12 and pytest 8.0.0 and now my testing panel look this:
![image](https://github.com/microsoft/vscode-python/assets/49996562/dbdf0eba-fd8d-46e7-aa79-b376131a6052)
What do I need to do to get it back?
Thanks

**Comment by RonnyPfannschmidt** [comment]
It's unclear what the problem is, I believe it's a bug in the testing panel 

Please report upstream

**Reply by Wintreist** [reply]
Sorry, what do you mean by "report upstream"?

**Reply by RonnyPfannschmidt** [reply, marked as answer]
The issue is with the pytest- 8 support of your dev tool

Pytest itself is unrelated to that and we have no detail knowledge how it is done

**Reply by Wintreist** [reply]
So, I'll declare it as a bug in vscode/python
